app/controllers/sms_controller.rb

```diff
@@ -0,0 +1,62 @@
+class SmsController < ApplicationController
```

Comment:
I think structurally it may make sense to nest this in a namespace with the Twilio voice controller, right? Maybe in a new namespace named `Twilio`?